web/__test__/components/ThemeSwitcher.test.ts

```
@@ -0,0 +1,274 @@
+        const currentTheme = 'azure';
+        const newTheme = 'black';
+        const wrapper = mount(ThemeSwitcher, {
+          props: { current: currentTheme },
+          global: { plugins: [createTestingPinia({ createSpy: vi.fn })] },
+        });
+        const select = wrapper.find('select');
+
+        // Mock the successful response callback chain
+        const mockResCallback = vi.fn();
+
+        mockFormUrlPostRes.mockImplementation((cb: Function) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)